__tests__/integration.tests.js

```diff
@@ -0,0 +1,41 @@
+const testData = require('../db/data/test-data/index.js');
+const request = require("supertest");
+const db = require("../db/connection");
+const { app } = require("../app.js");
+const seed = require("../db/seeds/seed");
+
+afterAll(() => {
+    db.end();
+});
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+
+describe('GET /api/topics', () => {
+    test('"should respond with 200 upon GET /api/topics" ', () => {
+        return request(app).get("/api/topics").expect(200);
+    });
+    test("GET: 200 should respond with an array of objects", () => {
```

Comment:
This test could be combined with the test below to make your code more dry. 
Don't forget to check the length of the array too as it should always be the same and we want to avoid automatically passing tests with an empty array.